subprojects/docs/src/docs/userguide/dep-man/03-controlling-transitive-dependencies/platforms.adoc

```diff
@@ -204,7 +204,7 @@ include::sample[dir="snippets/dependencyManagement/catalogs-settings/kotlin",fil
 ====
 Each catalog will generate an extension applied to all projects for accessing its content.
 As such it makes sense to reduce the chance of collisions by picking a name that reduces the potential conflicts.
-Gradle will emit a deprecation warning if a catalog does not have a name that ends with `Libs`.
+As an example, one option is to pick a name that ends with `Libs`.
```

Comment:
I think I preferred the original version of this statement (though I would change "will emit" to "emits"). As far as I can tell from the code, "Libs" is the *only* suffix you can use to avoid this warning. If there's a way to use other suffixes, I would use this sentence and perhaps provide an example of extending the syntax to another suffix below.  Otherwise, i think we should revert with the minor wording change I suggest above.